shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -774,6 +776,12 @@ object GlutenConfig {
       .booleanConf
       .createWithDefault(true)
 
+  val COLUMNAR_SHUFFLE_REALLOC_THRESHOLD =
+    buildConf("spark.gluten.sql.columnar.shuffle.realloc.threshold")
```

Comment:
nit: would you like copy docs from Configuration.md?

Reply:
@Yohahaha This PR depends on #2982 Please check this commit only https://github.com/oap-project/gluten/pull/3159/commits/feb200942a9f505bbf7394a4a1117040fa3748f4